cscs-checks/microbenchmarks/cpu/simd/benches/benches.hpp

```diff
@@ -0,0 +1,72 @@
+#ifndef BENCHES_HPP
```

Comment:
Is this file ours? Should it have the copyright header?

Reply:
it's from the [simd repo](https://github.com/agenium-scale/nsimd/blob/master/benches/benches.hpp)
What do you think is the best ?

Reply:
Perhaps we can add a single line comment pointing to the file on that repo.

Reply:
added a comment in the check